Packs/FortiGate/Integrations/FortiGate/README.md

```diff
@@ -1186,3 +1186,126 @@
 <p> </p>
 <h5>Command Example</h5>
 <pre>  !fortigate-get-banned-ips </pre>
+<h3 >19. Create an address object</h3>
```

Comment:
Please add a link to this command in the beginning of the file (lines 44)
![Screen Shot 2022-08-01 at 10 30 52](https://user-images.githubusercontent.com/78307768/182096410-d9c409cd-6040-4b94-a913-32213afe99eb.png)